programs/openbook-v2/src/instructions/settle_funds.rs

```diff
@@ -32,6 +32,16 @@ pub fn settle_funds<'info>(ctx: Context<'_, '_, '_, 'info, SettleFunds<'info>>)
 
     let seeds = market_seeds!(market, ctx.accounts.market.key());
 
+    if pa.penalty_events > 0 {
+        system_program_transfer(
+            pa.penalty_events * PENALTY_EVENT_HEAP,
```

Comment:
can just be a single penalty @binyebarwe don't think we need to charge per event